modules/nf-core/gatk4/cnnscorevariants/environment.yml

```diff
@@ -0,0 +1,8 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  # renovate: datasource=conda depName=bioconda/gatk4
+  - bioconda::gatk4=4.5.0.0
```

Comment:
should we note that cnnscore variants was not upgraded and point to the deprecation notice, and request to switch to the other tool?

Reply:
Good idea